capstone-ui/src/components/Login/Login.jsx

```diff
@@ -2,47 +2,51 @@ import * as React from "react";
 import "./Login.css";
 import axios from "axios";
 import * as config from "../../config";
+import { useEffect } from "react";
+import Loading from "../Loading/Loading";
+import LinkedinLogin from "../LinkedinLogin/LinkedinLogin";
 
-export default function Login({ handleLogin }) {
-  const username = React.createRef();
-  const password = React.createRef();
-
-  const handleSubmit = (event) => {
-    event.preventDefault();
-
-    const login = async () => {
-      try {
-        console.log("Logging in");
-        const res = await axios.post(`${config.API_BASE_URL}/login`, {
-          username: username.current.value,
-          password: password.current.value,
-        });
-        console.log("username: ", username.current.value);
-        handleLogin(res.data.user);
-      } catch (err) {
-        alert(err);
-        console.log(err);
-      }
-    };
-    login();
-  };
-
-  return (
-    <div>
-      <form onSubmit={handleSubmit}>
-        <div className="login">Login</div>
-        <label className="form-input">
-          <span>Username</span>
-          <input ref={username}></input>
-        </label>
-        <label className="form-input">
-          <span>Password</span>
-          <input type="password" ref={password}></input>
-        </label>
-        <button type="submit" className="button">
-          Login
-        </button>
-      </form>
+export default function Login({ onClickLogIn, onClickSignUp, isLoading }) {
+  return isLoading ? (
+    <Loading></Loading>
+  ) : (
+    <div className="login" id="login">
+      <h1>Login</h1>
+      <input
+        id="email"
```

Comment:
I think we need email, username and password as state, or refs, in order to be able to pass them to `onClickLogIn`